### ISSUE TITLE
"node create-channel.js" fails

### ISSUE THREAD

**Issue by rupc** [sujayv/basic-blockchain-app#1]
Hi. 
I followed your instructions, but I got the following error message at "node create-channel.js" step.

> not ok 3 TypeError: _commonProto.Envelope.decode is not a function at Client.createChannel (/home/jyr/work/proj/basic-blockchain-app/node_modules/fabric-client/lib/Client.js:227:40) at hfc.newDefaultKeyValueStore.then.then (/home/jyr/work/proj/basic-blockchain-app/create-channel.js:84:17)
     operator: fail
    at: hfc.newDefaultKeyValueStore.then.then.then (/home/jyr/work/proj/basic-blockchain-app/create-channel.js:109:5)
  ...

Here is problematic code lines

**create-channel.js**
```javascript
var hfc = require('fabric-client');
...
var client = new hfc();
...
var request = {
	envelope : data,
	name : 'mychannel',
	orderer : orderer
};
// send to orderer
return client.createChannel(request);
```

**fabric-client/lib/Client.js**
```javascript
var _commonProto = grpc.load(__dirname + '/protos/common/common.proto').common;
...
var envelope = _commonProto.Envelope.decode(request.envelope);
```
I'm using hyperledger/fabric-peer:x86_64-1.0.0-alpha, hyperledger/fabric-ca:x86_64-1.0.0-alpha, hyperledger/fabric-orderer:x86_64-1.0.0-alpha 

I didn't study grpc in nodejs yet, but it seems that grpc code causes the problem... 
It would be appreciate if you explain why this error occurs.

**Comment by sujayv** [comment]
It seems to be an issue due to the latest version of the fabric images. Did you clone the repository at the commit number in the README?

**Comment by rupc** [comment]
That was my mistake. But even though I switched to specified commit hash and make docker images from that, same issue still remains.
One thing to be noted is that while you include 'fabric-client' at the top directory(basic-blockchain-app/fabric-client), "npm install" command also creates 'fabric-client' in node_modules directory. And "create-channel.js" seeks 'fabric-client' in node_modules, not already existing 'fabric-client'
So I compare each size of 'fabric-client' at different locations. 
```
du -sh fabric-client/
572K	fabric-client/

du -sh node_modules/fabric-client/
912K	node_modules/fabric-client/
```
Definitely, It shows those two have different content. 
If grpc code is not the real problem, fabric-client module your program actually uses might not be the intended version.

---
I moved basic-blockchain-app/fabric-client to basic-blockchain-app/node_modules/ and test it.
But still "not ok 3 TypeError: _commonProto.Envelope.decode is not a function at Client.createChannel" error comes in. 

What should I do to fix?
